doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -20018,26 +20022,28 @@ where
 
 #### Field: Name
 
-This field signifies a unique identifying name in a group referring GenericContaminanteSourceAndSinkNames.
+This field signifies a unique identifying name.
 
 #### Field: Surface Name
 
 This field denotes the name of the surface as a generic contaminant reversible source or sink using the boundary layer diffusion model.
 
-#### Field: Film Transfer Coefficient
+#### Field: Mass Transfer Coefficient
 
 This field specifies the average mass transfer coefficient of the contaminant generic contaminant within the boundary layer (or film) above the surface of the adsorbent.
 
-#### Field: Henry Partition Coefficient
+#### Field: Schedule Name
 
-This field denotes the generic contaminant Henry partition coefficient in the units of dimensionless. The coefficient relates the concentration of the contaminant generic contaminant in the bulk-air to that at the surface of the adsorption material.
+This field is the name of the schedule (ref: Schedule) that modifies the mass transfer coefficient with the value between 0.0 and 1.0. 
 
+#### Field: Henry adsorption constant or partition coefficient
```

Comment:
@mjwitte and @Myoldmopar I change IO first and will make idd changes after the release.At the same time, I am going to post an issue.